Chronological/QueryResults/Events/EventQueryResultToTypeMapper.cs

```diff
@@ -62,6 +62,14 @@ internal IEnumerable<T> Map<T>(IEnumerable<EventResult> eventResults)
                                 }
                             }
                         }
+                        else if (typeProperty.CanWrite && attributes.Any(x => x.EventFieldName == BuiltIn.EventTimeStamp))
+                        {
+                            if (propertyType.ToLower() == "datetime" &&
+                                (typeProperty.PropertyType == typeof(DateTime) || typeProperty.PropertyType == typeof(DateTime?)))
+                            {
+                                typeProperty.SetValue(instance, DateTime.Parse(value));
```

Comment:
The parsing will throw an exception if `value` is null. I don't know if you make this on purpose but I think it can be dangerous

Reply:
the code above does the same and ti does not look like an issue. but i will optimize it anyway

Reply:
see commit below